ISISComputingGroup/ibex_gui
# Ticket 4019: Monitor block units

### Description of work

Monitors the units separately and appends them on the end of the block value.

### Ticket

See https://github.com/ISISComputingGroup/IBEX/issues/4019

### To test

* Start up a configuration with a block on a PV
* Change the EGU field of the PV and confirm the blocks view updates
* Create a synoptic with a component pointing to the same PV
* Change the EGU field of the PV and confirm the synoptic view updates view updates
* Do the above pointing at a PV that has no units (e.g. an MBBI) and confirm that you still get a sensible value back

### Unit tests

Added tests.

---

#### Code Review


### Final Steps
